narratives/ncov_sit-rep_2020-05-15.md

```diff
@@ -0,0 +1,220 @@
+<br><br>
+Importantly, the vast majority of these mutations either "break" the virus such that it can't transmit and/or replicate anymore, or don't change the virus at all because of [redundant encoding](https://en.wikipedia.org/wiki/Synonymous_substitution).
+Rarely, a mutation can help the virus replicate and/or transmit better, but nearly all of these changes still have only a tiny effect.
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Viral mutations are useful for tracing outbreaks](https://nextstrain.org/ncov/global/2020-05-14?c=gt-ORF3a_57&d=tree,entropy&f_division=New%20York&m=div&p=full)
+
+These mutations can be used as "bread crumbs" to trace the history of a given sample. Just as the people in your family share unique combinations of genetic characteristics, closely related viral samples share unique combinations of genetic mutations.
+<br><br>
+For example, here we show the viral "family tree" with each sample colored by which amino acid is present at a specific location in the viral genome (ORF1a, site 57). Highlighting samples from New York, we see that most of the cases from New York and Europe have the amino acid Histidine ("H") rather than Glutamine ("Q") at this position. This, in combination with all the other sites in the genome, tells us that these cases are all closely related to each other.
+<br><br>
+For a more in-depth explanation of these concepts, we recommend [this visual explanation](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html) by Jonathan Corum and Carl Zimmer.
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [The mutation rate of SARS-CoV-2 is typical for a coronavirus](https://nextstrain.org/ncov/global/2020-05-14?c=num_date&d=tree,entropy&l=clock&p=full)  
+
+Because mutations are biologically inevitable, they tend to accumulate at a steady rate over time.
+<br><br>
+Here, the x axis shows the date that each sample was collected. The y axis shows the *total number* of mutations away from the root of the tree. Each sample is colored by which region of the world it was collected in.
+<br><br>
+While there are certainly some outliers, on average, we see changes accumulate at a rate of ~24 substitutions per year. This means that if a single viral lineage were transmitting from one person to the next for a full year, we'd expect it to accumulate roughly 24 substitutions by the end of the year. Importantly, this corresponds to only (24/30,000)\*100\% = ~0.08\% of the viral genome changing each year.
+<br><br>
+For context, SARS (also a coronavirus) accumulates XX substitutions per year (changing XX\% of its genome), while influenza accumulates XX substitutions per year (changing XX\% of its genome).
```

Comment:
SARS probably not a good example. It's basically the same as SARS-CoV-2 (all the coronaviruses are). So your above numbers will basically work here...
Flu also tricky because it's much smaller than coronavirus. So it also has ~25 muts per year & ~2 per month (per genome) but as the genome is 13-14k, that translates to ~0.16\% genome change (guessing - didn't actually do the math). 

In the FT piece I helped with, I gave things as mutations per 1,000 bases in a year because it avoided this problem - but it's obviously, clearly nonstandard. Here's what I told them: "On average, SARS-CoV-2 would have one mutation every 1000 bases in a year, whereas Flu would have on average 2 mutations every 1000 bases. On the same scale, HIV would have on average 4 mutations every 1000 bases in a year."

Reply:
This is fantastic, thank you Emma.